cli/addmfadevice.go

```diff
@@ -0,0 +1,83 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/99designs/keyring"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/99designs/aws-vault/vault"
+)
+
+type AddMfaDeviceCommandInput struct {
+	ProfileName  string
+	Keyring      keyring.Keyring
+	Config       vault.Config
+	Username     string
+	RequireTouch bool
+}
+
+func ConfigureAddMfaCommand(app *kingpin.Application) {
+	input := AddMfaDeviceCommandInput{}
+
+	cmd := app.Command("add-mfa-device", "Adds a MFA device to the IAM user")
+	cmd.Arg("username", "Name of the user to add the MFA as device for").
+		Required().
+		StringVar(&input.Username)
+
+	cmd.Arg("profile", "Name of the profile").
+		Required().
+		HintAction(awsConfigFile.ProfileNames).
+		StringVar(&input.ProfileName)
+
```

Comment:
Should we flip this to `add-mfa-device <profile> <username>`, as all the other commands have profile as the first argument